autonomy/deploy/generators/docker_compose/templates.py

```diff
@@ -29,8 +29,7 @@
 
 DOCKER_COMPOSE_TEMPLATE: str = """version: "2.4"
 services:
-{tendermint_nodes}
-{abci_nodes}
+{hardhat_node}{acn_node}{tendermint_nodes}{abci_nodes}
```

Comment:
Previously multi-line, now single line. Does it matter?

Reply:
Not much, just for better formatting in the final template